src/bin/lockin-external.rs

```diff
@@ -117,8 +117,14 @@ const APP: () = {
         let iir_state = c.resources.iir_state;
         let lockin = c.resources.lockin;
 
+        let t = c
+            .resources
+            .timestamper
+            .latest_timestamp()
+            .unwrap_or_else(|t| t) // Ignore timer capture overflows.
```

Comment:
Are we currently hitting cases where the capture is overflowing? I feel a tad uncomfortable with silently ignoring these faults, since they _should_ be indicating that something isn't working how it should.

Reply:
They are hit easily in practice when playing with the device by either glitchy inputs or fast input frequency. Electrical inputs should never panic/crash the firmware. In all cases, especially this one, the DSP/code will recover once the source of the invalid input is removed.

Reply:
Would you mind adding a TODO here to increment some counter value if we hit a fault? That would at least remind us to report these failures via telemetry so we can diagnose a faulty connection or input in the future.

Reply:
There is no TODO needed here. A timer overflow has the similar consequences and the same severity as a voltage clipping or some signal hitting a limit.

Reply:
I'm simply trying to say it would be a useful metric to at least inform the user that timestamps are overflowing when we don't expect them to. Otherwise, it may be difficult to pinpoint faults

Reply:
Signalling that the reference clock is fast or slow or the input voltage is high or low is nice. Those are parts of the bigger topic of monitoring. There are a good dozen equally important other places. They can be readily identified. I wouldn't sprinkle TODOs unless we have a clear plan or desire to signal but trouble finding them.